slack_bolt/context/base_context.py

```diff
@@ -103,13 +105,23 @@ def matches(self) -> Optional[Tuple]:
         """Returns all the matched parts in message listener's regexp"""
         return self.get("matches")
 
+    @property
+    def function_execution_id(self) -> Optional[str]:
+        """The `function_execution_id` associated with this request."""
+        return self.get("function_execution_id")
+
     # --------------------------------
 
     @property
     def authorize_result(self) -> Optional[AuthorizeResult]:
         """The authorize result resolved for this request."""
         return self.get("authorize_result")
 
+    @property
+    def function_bot_access_token(self) -> Optional[str]:
+        """The bot token resolved for this function request."""
```

Comment:
Same as above re: only available for `function_executed` events and interactivity events scoped to a function.